lib/src/blocs/i_movies_bloc.dart

```diff
@@ -5,4 +5,8 @@ abstract class IMoviesBloc extends IBloc {
   Stream<Movie> get streamMovies;
 
   void fetchMoviesApi();
+
+  void fetchMoviesFilter(
+    String filterKeys,
```

Comment:
You can change this parameter name to query instead of filterKeys

Reply:
done